rai/node/node.cpp

```diff
@@ -1285,9 +1285,9 @@ void rai::block_processor::process_receive_many (std::unique_lock<std::mutex> &
 	// Processing blocks
 	while ((!blocks.empty () || !forced.empty ()) && std::chrono::steady_clock::now () - start_time < node.config.block_processor_batch_max_time)
 	{
-		if (blocks.size () > 64 && should_log ())
+		if ((blocks.size () + state_blocks.size ()) > 64 && should_log ())
 		{
-			BOOST_LOG (node.log) << boost::str (boost::format ("%1% blocks in processing queue") % blocks.size ());
+			BOOST_LOG (node.log) << boost::str (boost::format ("%1% blocks in processing queue") % (blocks.size () + state_blocks.size ()));
```

Comment:
Maybe for now, until we do merge in #1450:

```suggestion
			BOOST_LOG (node.log) << boost::str (boost::format ("%1% (+ %2%) blocks in processing queue") % blocks.size () % state_blocks.size ());
```